metrics/metrics/zombie_instances.go

```diff
@@ -30,6 +30,14 @@ func (zi *ZombieInstances) Columns() []Column {
 	return zi.columns
 }
 
+func (*ZombieInstances) Type() MetricType {
+	return TimeBasedMetric
+}
+
+func (*ZombieInstances) RelevantDelta() int {
```

Comment:
Can we use the https://golang.org/pkg/time/#Duration type here?

Reply:
The problem is that for BuildBasedMetric instances RelevantDelta() returns a number of changelists.

For TimeBasedMetric this value is later converted to a time.Duration object.